docs/blocks/machines/alloy_smelter.mdx

```diff
@@ -17,16 +17,128 @@ The **Alloy Smelter** is a machine that creates metal alloys from ingots or dust
 
 <CraftingTable recipe="input air techreborn:electronic_circuit air input techreborn:electric_furnace techreborn:iron_alloy_furnace techreborn:electric_furnace input air air air output techreborn:alloy_smelter"/>
 
-### List of alloys
+### Crafting Recipes
+		<details>
+			<summary>Recipes using Alloy Smelter</summary>
+			<Machine config={{
```

Comment:
Will it be simplier to have something like
<Machine recipe="aloy_smelter/brass_ingot" />  

and do not copy content of recipes into machine page?